flow/shared/alerting/slack_alert_sender.go

```diff
@@ -0,0 +1,38 @@
+package alerting
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/slack-go/slack"
+)
+
+type slackAlertSender struct {
+	client     *slack.Client
+	channelIDs []string
+}
+
+type slackAlertConfig struct {
+	AuthToken  string   `json:"auth_token"`
+	ChannelIDs []string `json:"channel_ids"`
+}
+
+func newSlackAlertSender(config *slackAlertConfig) *slackAlertSender {
+	return &slackAlertSender{
+		client:     slack.New(config.AuthToken),
+		channelIDs: config.ChannelIDs,
+	}
+}
+
+func (s *slackAlertSender) sendAlert(ctx context.Context, alertTitle string, alertMessage string) error {
+	for _, channelID := range s.channelIDs {
```

Comment:
can you send `PEERDB_DEPLOYMENT_UID`  as well?

Reply:
done